### PR TITLE
adding more env vars

### PR DESCRIPTION
**Pull request by Gerson2102**
Added more env vars to the Readme.md file.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -143,16 +143,34 @@ $ LIBRARY_PATH="[gcc-path value]" LD_LIBRARY_PATH="[gcc-path value]" rustc +$(ca
...
 <dl>
-    <dt>CG_GCCJIT_INCR_CACHE_DISABLED</dt>
-    <dd>Don't cache object files in the incremental cache. Useful during development of cg_gccjit
-    to make it possible to use incremental mode for all analyses performed by rustc without caching
-    object files when their content should have been changed by a change to cg_gccjit.</dd>
+    <dt>CG_GCCJIT_DUMP_ALL_MODULES</dt>
+    <dd>Enables dumping of all compilation modules. When set to "1", a dump is created for each module during compilation and stored in `/tmp/reproducers/`.</dd>
+    <dt>CG_GCCJIT_DUMP_MODULE</dt>
+    <dd>Enables dumping of a specific module. When set with the module name, e.g., `CG_GCCJIT_DUMP_MODULE=module_name`, a dump of that specific module is created in `/tmp/reproducers/`.</dd>
     <dt>CG_GCCJIT_DISPLAY_CG_TIME</dt>
```
Comment (antoyo) [inline Readme.md:150]: This seems unused, so please remove.

File: Readme.md [hunk 2]
```diff
@@ -143,16 +143,34 @@ $ LIBRARY_PATH="[gcc-path value]" LD_LIBRARY_PATH="[gcc-path value]" rustc +$(ca
 ## Env vars
 
 <dl>
-    <dt>CG_GCCJIT_INCR_CACHE_DISABLED</dt>
-    <dd>Don't cache object files in the incremental cache. Useful during development of cg_gccjit
-    to make it possible to use incremental mode for all analyses performed by rustc without caching
-    object files when their content should have been changed by a change to cg_gccjit.</dd>
+    <dt>CG_GCCJIT_DUMP_ALL_MODULES</dt>
```
Comment (GuillaumeGomez) [inline Readme.md:146]: @antoyo: We should really remove this HTML and just write plain markdown...

**Review by antoyo — COMMENTED** [review]
Thanks! One variable is to remove.

File: Readme.md (hunk 2 above)
Comment (antoyo) [reply Readme.md:146]: Please suggest the formatting to be used.

File: Readme.md (hunk 1 above)
Comment (antoyo) [reply Readme.md:150]: The variables you removed are used.
You should add them back and remove `CG_GCCJIT_DISPLAY_CG_TIME` instead.

**Review by antoyo — APPROVED** [review]
I'm waiting for @GuillaumeGomez answer about the Markdown vs HTML preference.
Good job!

**Comment by antoyo** [comment]
Thanks for your contribution!

**Comment by Gerson2102** [comment]
@antoyo thanks for the opportunities :)

**Pull request merged**